service/src/main/kotlin/fi/espoo/evaka/emailclient/IEmailMessageProvider.kt

```diff
@@ -90,6 +91,11 @@ interface IEmailMessageProvider {
         language: Language,
         events: List<CalendarEventNotificationData>
     ): EmailContent
+
+    fun financeDecisionNotification(
+        language: Language,
```

Comment:
`language` is not used, so it should be removed from the signature. We could remove the `language` parameter from all templates in a future refactor.

Reply:
Removed.